hugegraph-core/src/main/java/com/baidu/hugegraph/structure/HugeVertex.java

```diff
@@ -283,6 +283,7 @@ public HugeEdge constructEdge(String label, HugeVertex vertex,
 
         // Check target vertex
         E.checkArgumentNotNull(vertex, "Target vertex can't be null");
+        // TODO: Why we check it? the param is HugeVertex already?
```

Comment:
maybe the signature of constructEdge changed from Vertex to HugeVertex

Reply:
> changed from Vertex to HugeVertex? 
do u means  "changed from HugeVertex to Vertex" ?  

so shall we consider **comment the code** instead of check it in a main logic now?